### DISCUSSION TITLE
Next First Gejala pada halaman Admin

### DISCUSSION THREAD

**Discussion by 4njas** [rizkyilhampra/spdhtc#74] [Q&A] [answered]
Hallo bang, mau nanya mengenai kolom next firs gejala, setelah semua rule saya hapus dan saya input gejala yang baru, saya coba bikin rule baru, kenapa list pada kolom next first gejala tidak muncul drop list gejalanya, ketika di klik hanya muncul teks "pilih gejala selanjutnya", tidak ada pilihan lain? sedangkan dikolom gejala, muncul drop list gejala yang bisa dipilih.

![WhatsApp Image 2024-06-26 at 00 24 49_711f4bdf](https://github.com/rizkyilhampra/spdhtc/assets/86731218/6ba1257a-50c4-4ae5-b780-cfacf4ed9934)
![WhatsApp Image 2024-06-26 at 00 24 49_6d370e27](https://github.com/rizkyilhampra/spdhtc/assets/86731218/9b3e054b-4135-4eaa-9be0-a3d11b0f839e)

**Comment by rizkyilhampra** [comment, marked as answer]
Hallo. 

FYI mungkin ini juga jadi salah satu alasan mengapa SPDHTC versi `v2.*` atau versi terakhir masih termasuk ke dalam *beta release*. 

Jadi untuk di versi `v2.*`, tidak bisa untuk membuat aturan melalui web. Hanya bisa melalui mekanisme *seeding*. Untuk *seeder* yang berkaitan dengan aturan adalah file berikut 

https://github.com/rizkyilhampra/spdhtc/blob/master/database/seeders/NextFirstGejalaSeeder.php 
https://github.com/rizkyilhampra/spdhtc/blob/master/database/seeders/Rule.php

Untuk keseluruhan *seeder* dapat lihat di *folder* dengan path `database/seeders`. Untuk informasi lainnya dapat dilihat di sini https://laravel.com/docs/9.x/seeding

Terima kasih
